isotovideo

```diff
@@ -39,6 +39,21 @@ use Mojo::Base -strict, -signatures;
 use autodie ':all';
 no autodie 'kill';
 
+# Avoid JSON::PP::Boolean warnings
+#
+# When Cpanel::JSON::XS is loaded before JSON::PP, it will
+# result in warnings
+#
+#     Subroutine JSON::PP::Boolean::(0+ redefined at .../overload.pm
+#     ...
+#
+# This is because Cpanel::JSON::XS creates the same overloads
+# as JSON::PP. This is handled in newer versions of JSON::PP, so
+# it will go away at some point, but until then it is always confusing to see
+# the warnings.
+# Loading it here first will avoid the warning.
```

Comment:
I suggest to make the comment at most two-line, e.g. the subject "Avoid…" and a reference to any bug or ticket reference for details.